scripts/backfill_cdc.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Backfills all the CDC tables
+set -e
+
+declare -a STORAGES=("groupedmessages"  "groupassignees")
+
+log_message() {
+    GREEN='\033[0;32m'
+    NC='\033[0m'
+
+    echo -e "$GREEN"
+    echo "$1"
+    echo -e "$NC"
+}
+
+mkdir -p /tmp/cdc-snapshots/
+
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+log_message "********* Taking the snapshot from Postgres *********"
+
+cd "$(dirname "$0")"
+
+docker run \
+-v "$(pwd)"/../config/cdc/configuration.yaml:/etc/cdc/configuration.yaml \
+-v "$(pwd)"/../config/cdc/cdc-snapshot-config.yaml:/etc/cdc/cdc-snapshot-config.yaml \
+-v /tmp/cdc-snapshots:/tmp/cdc-snapshots \
+--rm \
+--network sentry \
+getsentry/cdc:nightly \
+cdc -c /etc/cdc/configuration.yaml \
+snapshot --snapshot-config /etc/cdc/cdc-snapshot-config.yaml \
+2>&1 | tee /tmp/cdc-snapshots/snapshot.log
+
+SNAPSHOT_ID=$(awk '{ if($4=="Starting" && $5=="snapshot" && $6=="ID") print $7}' /tmp/cdc-snapshots/snapshot.log )
```

Comment:
You can also do this if you have the image name:
`docker ps -qf "name=^name_of_image$`

Reply:
I am not sure I follow. This code is meant to find an id generated when running the container `snapshot` command. Which identifies a file name the command at line 47 needs to load. 
isn't `docker ps -qf "name=^name_of_image$` returning the container id? Which is not what I am trying to load.

Specifically I am looking for this line in the log (which is the output of the command above)
```
2021-04-27 06:05:30,833 INFO     Starting snapshot ID 922ed908-a71e-11eb-98bb-0242ac120008
```

Reply:
My apologies! I misread this.

I'm mainly concern if the formatting of that line was to change and not match.

What would be consequences of SNAPTSHOT_PATH having the wrong SNAPSHOT ID?

Reply:
The second command will raise an error and stop.
I am not too concerned about this as of now as this is only for  the dev environment. 

I can improve this by letting the user pick the destination directory in the first script. But I would have to change that script as well, and that would take a while